lib/query-builders/admin.js

```diff
@@ -23,7 +24,8 @@ module.exports = {
       .where(buildEstablishmentQuery(profile))
       .whereIn('status', [
         returnedToApplicant.id,
-        recalledByApplicant.id
+        recalledByApplicant.id,
+        withAdmin.id
```

Comment:
I would probably not put this in here and would instead add a separate query builder for endorsement that applies to anyone who meets the `canEndorse` criteria.

`./lib/query-builders/endorse.js`

Reply:
🌮